src/DssSpell.t.sol

```diff
@@ -911,28 +951,230 @@ contract DssSpellTest is DssSpellTestBase {
     }
 
     // SPELL-SPECIFIC TESTS GO BELOW
+    bytes32           constant  SRC_ILK       = "PSM-USDC-A";
+    bytes32           constant  DST_ILK       = "LITE-PSM-USDC-A";
+    PsmAbstract       immutable srcPsm        = PsmAbstract(      addr.addr("MCD_PSM_USDC_A"));
+    LitePsmLike       immutable dstPsm        = LitePsmLike(      addr.addr("MCD_LITE_PSM_USDC_A"));
+    GemAbstract       immutable gem           = GemAbstract(      addr.addr("USDC"));
```

Comment:
nit: trailing spaces

Reply:
this is a "feature", it is supposed to be formatting, like here for example: https://github.com/makerdao/spells-mainnet/blob/master/src/DssSpell.t.base.sol#L241